src/pages/events.js

```diff
@@ -220,92 +222,96 @@ const EventPage = ({
   const eventStructuredData = createEventStructuredData(futureEvents)
 
   return (
-    <Layout
-      bgColor="blueBg"
-      footerContactUsId={footerContactUs.id}
-      breadcrumbData={breadcrumbData}
-    >
-      <Helmet>
-        {eventStructuredData &&
-          eventStructuredData.length > 0 &&
-          eventStructuredData.map(data => (
-            <script key={generate()} type="application/ld+json">
-              {JSON.stringify(data)}
-            </script>
-          ))}
-      </Helmet>
-      <Head seoMetaData={seoMetaData} />
-
-      <StyledBlueBackground>
+    <LogoStyleContext.Provider value="white">
```

Comment:
No one can read a diff like this, I assure that I've only wrapped it in the `Provider` :joy:

Reply:
Aha I trust you!